packages/components/src/internal/components/ontology/OntologyBrowserPanel.tsx

```diff
@@ -21,6 +22,29 @@ export interface OntologyBrowserProps {
     filterChangeHandler?: (filter: PathModel) => void;
 }
 
+export const OntologyBrowserPage: FC<OntologyBrowserProps> = memo( props => {
+    const {initConceptCode, ...rest} = props;
+    const [concept, setConcept] = useState<ConceptModel>();
+    const [loading, setLoading] = useState<boolean>();
+
+    useEffect(() => {
+        if (initConceptCode && !concept) {
+            setLoading(true);
+            (async () => {
+                const loadingConcept = await fetchConceptForCode(initConceptCode);
+                setConcept(loadingConcept);
+                setLoading(false);
+            }) ();
```

Comment:
We should catch the error case where the concept cannot be found. I tested this locally by editing the URL to be a concept code that didn't exist and my page just stays in the loading spinner state. As an example for how to display / handle the error case, you can look at the `OntologyBrowserPanel` where if you give it an invalid `ontologyId` param it shows the error and falls back to the display without that param.